vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -598,6 +601,19 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
         vmConfig.setGuestId(guestOsIdentifier);
     }
 
+    public static void addUSBController(VirtualMachineConfigSpec vmConfigSpec) {
```

Comment:
@sateesh-chodapuneedi all the prepareXyzDevice() methods in VmwareHelper class returns VirtualDevice. So to be consistent, i think it is better to return a VirtualDevice when preparing USB controller and this device is added to the device config spec same as other devices.